source/Ship.h

```diff
@@ -366,8 +366,8 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	CargoHold &Cargo();
 	const CargoHold &Cargo() const;
 	// Display box effects from jettisoning this much cargo.
-	void Jettison(const std::string &commodity, int tons);
-	void Jettison(const Outfit *outfit, int count);
+	void Jettison(const std::string &commodity, int tons, bool wasDumped = false);
+	void Jettison(const Outfit *outfit, int count, bool wasDumped = false);
```

Comment:
Given that wasDumped is technically inaccurate now (as the player can dump cargo but not have the flotsam's government be set), should this variable be renamed? Would something like saveGov or recordGov be more appropriate?

Reply:
renamed to wasAppeasing to reflect the intention of the ship that was dumping the cargo.